src/parsers/vc12DXMLLoad.cpp

```diff
@@ -269,31 +311,47 @@ void vc12DXML_Project::SetName(const char *pName)
   m_pName = udStrdup(pName);
 }
 
-udResult vc12DXML_Project::BeginBuild(udJSON const *pNode)
-{
-  return Build(pNode, m_globals);
-}
-
-udResult vc12DXML_Project::Build(udJSON const *pNode, vc12DXML_ProjectGlobals &globals)
+udResult vc12DXML_Project::AddProject(udJSON const *pNode, vc12DXML_ProjectGlobals &globals)
```

Comment:
`globals` should be `const &` or `*pGlobals`, you either don't modify it, or you make it clear in the calling code that the value is being modified. `AddProject(..., globals);` vs `AddProject(..., &globals);`, in the second example it's clear that we expect `globals` to be modified, but we don't know in the first example if it's being modified or not - if we avoid passing by reference (unless it's const) then we do know.